src/delete/delete_function.py

```diff
@@ -2,13 +2,24 @@
 import boto3
 import constants
 import custom_exceptions
+import logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
 
 def lambda_delete_dbinstance(event, context):
     """Handles deletion of a RDS db instance"""
     region = os.environ['Region']
     rds = boto3.client('rds', region)
     result = {}
     instance_id = event['identifier'] + constants.TEMP_POSTFIX
+
+    logger.info('## ENVIRONMENT VARIABLES')
+    logger.info(region)
+    logger.info('## EVENT')
+    logger.info(event)
+    logger.info('## INSTANCE ID')
+    logger.info(instance_id)
     try:
         rds.delete_db_instance(
```

Comment:
add logs for starting of the particular task execution and similarly at the completion.